substra/sdk/backends/remote/rest_client.py

```diff
@@ -24,13 +24,13 @@
 class Client():
     """REST Client to communicate with Substra server."""
 
-    def __init__(self, url, version, insecure, token):
+    def __init__(self, url, insecure, token):
         self._default_kwargs = {
             'verify': not insecure,
         }
         self._headers = {
             'Authorization': f"Token {token}",
-            'Accept': f'application/json;version={version}',
+            'Accept': 'application/json',
```

Comment:
Did you try running substra-tests with this code? I think it should fail because the version is missing. You should instead write `'Accept': 'application/json;version=0.0',`

Reply:
I'll fix this.